components/Common/BlogPostCard/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { useMemo } from 'react';
+import type { ComponentProps, FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+
+import styles from './index.module.css';
+
+const dateTimeFormat = new Intl.DateTimeFormat(navigator.language, {
+  day: 'numeric',
+  month: 'short',
+  year: 'numeric',
+});
+
+type Author = {
+  fullName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
```

Comment:
Isn't `src` just a string? This sounds a mouthful of a type 😅

Reply:
I'm with you on this, thanks for pointing it out. I will set `src` to a `string`.